package.json

```diff
@@ -2,6 +2,9 @@
   "name": "com.testfairy.cordova-plugin",
   "version": "2.48.0",
   "description": "TestFairy SDK plugin for Cordova",
+  "engines": {
+    "node": ">=6.0.0"
```

Comment:
this simply matches what [cordova had with 9.0.0](https://github.com/apache/cordova-cli/blob/9.0.0/package.json#L7) to avoid any potential issues.